OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -440,10 +448,12 @@ protected override MovePart OnComplete(Actor self, Mobile mobile, Move parent)
 					parent.path.Add(nextCell.Value.First);
 				}
 
+				var toPos = mobile.ToCell.Layer == 0 ? map.CenterOfCell(mobile.ToCell) :
+					((ActorMap)self.World.ActorMap).CustomMovementLayers[mobile.ToCell.Layer].CenterOfCell(mobile.ToCell);
```

Comment:
`ActorMap` depends on `ICustomMovementLayer` which depends on `MobileInfo` (in Mods.Common), but `World.ActorMap` lives in Game.  This was the less bad alternative to querying MobileInfo from the actor for every pathfinder expand.